flink-table/flink-table-code-splitter/src/main/antlr4/JavaLexer.g4

```diff
@@ -0,0 +1,184 @@
+/**
```

Comment:
The original BSD license has been removed. I think this violates the licensing terms.